### PR TITLE
bugfix/CDAP-19971: hide view and restore options for "-SNAPSHOT" versioned pipeline

### PR DESCRIPTION
**Pull request by de-lan**
Hide view and restore options for "-SNAPSHOT" versioned pipeline

## PR Type
- [ ] Bug Fix
- [x] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19971](https://cdap.atlassian.net/browse/CDAP-19971)

### CODE REVIEW

**Comment by sumengwang** [comment]
Did we discuss why not just hide the entire row?

**Comment by de-lan** [comment]
> Did we discuss why not just hide the entire row?

The pagination might break?

File: app/cdap/components/PipelineHistory/PipelineHistoryTableRow.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ interface IPipelineHistoryTableRowProps {
 }
 
 const PREFIX = 'features.PipelineHistory.table';
+const SNAPSHOTVERSION = '-SNAPSHOT';
```
Comment (sumengwang) [inline app/cdap/components/PipelineHistory/PipelineHistoryTableRow.tsx]: What do you think adding this to `global-constants.js`?

Comment (de-lan) [reply app/cdap/components/PipelineHistory/PipelineHistoryTableRow.tsx]: sounds good! I've added it to global-constants

**Review by sumengwang — APPROVED** [review]

**Review by sanjanasandeep — APPROVED** [review]

**Review by radhikav1 — APPROVED** [review]

**Pull request merged**